src/components/AgreementSections/InfoPeriods.js

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  ButtonGroup,
+  Col,
+  FormattedUTCDate,
+  Headline,
+  KeyValue,
+  Layout,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+
+const InfoPeriods = ({ periods }) => {
+  /*
+   * This could be handled by callback, but since we need to know for display whether next/prev/current exist
+   * it seems more prudent to just fetch all three at once and then do our logic at this stage
+   */
+
+  const displayPeriodOptions = periods?.reduce((map, period) => {
```

Comment:
If we default the `periods` to be an empty array, we dont need the `?` and we can get rid if the default `{}` as well

Reply:
Sure